cheatsheets/JWT_Cheat_Sheet.md

```diff
@@ -0,0 +1,227 @@
+# JWT Cheat Sheet
+
+## Introduction
+
+Many applications use **JSON Web Tokens** (JWT) to allow the client to indicate its identity for further exchange after
```

Comment:
"to allow the *client*", I guess you mean the *user*. JWT can definitely be used to represent claims about the client, I think you are actually thinking more about the user.